pkg/commands/git_test.go

```diff
@@ -551,7 +552,7 @@ func TestGitCommandUpstreamDifferentCount(t *testing.T) {
 		{
 			"Can't retrieve pushable count",
 			func(string, ...string) *exec.Cmd {
-				return exec.Command("exit", "1")
+				return exec.Command("test")
```

Comment:
does test just return an exit status of 1?

Reply:
yep I was wrong with exit as it's a built-in shell command, it was working because exit wasn't found in path and so it was triggering an error

Reply:
haha that's interesting. I suppose with this we assume that 'test' isn't in path, which for now is a reasonable assumption

Reply:
test is a real command and when you invoke it without any comparison, you get an exit 1.

Reply:
oh, that's much better then :)

Reply:
yep